### ISSUE TITLE
Refactor Shedules

### ISSUE THREAD

**Issue by vitusbenson** [earthnet2021/earthnet-models-pytorch#29]
Currently, learning rate shedules are handled by just passing options to the generic PyTorch shedulers (https://github.com/earthnet2021/earthnet-models-pytorch/blob/main/earthnet_models_pytorch/task/spatio_temporal.py#L101) and other shedulers for model specific sheduling like dropout etc. are implemented seperately (https://github.com/earthnet2021/earthnet-models-pytorch/blob/main/earthnet_models_pytorch/task/shedule.py). This is confusing and can be improved.

Goal:
- Clearly document learning rate sheduling, since the code offers really high functionality, but given it is just one line and looks maybe a bit magical, it would be good to add documentation for this
- Improve sheduling API for model specific shedules <- should this still be part of Task? or should this be moved into models? (Can it even?, need steps for shedule in the end...)
